exercises/HangMan Game/HangMan.py

```diff
@@ -0,0 +1,69 @@
+# Variablen
+
+import zufallsworte as zufall
+
+wort = zufall.zufallswoerter(1)[0]  # gibt ein Zufallswort zurück
+
+
+gesucht = wort
+
+
+gefunden = []
+falsch_geraten = []
+
+
+def show():
+    print('Falsche Buchstaben:', falsch_geraten)
+    for buchstabe in gesucht:
+        if buchstabe in gefunden:
+            print(buchstabe, end=' ')
+        else:
+            print('_', end=' ')
+    print('')
+
+
+def is_valid(inp):
+    return True
+
+
+def eingabe():
+    buchstabe = input('Buchstabe? ')
+    while not is_valid(buchstabe):
+        buchstabe = input('Buchstabe? ')
+    return buchstabe.lower()
+
+
+def auswerten(valid_inp):
+    if valid_inp in gesucht:
+        gefunden.append(valid_inp)
+    else:
+        falsch_geraten.append(valid_inp)
+
+
+def gewonnen():
+    return False
+
+
+def game_over():
+     
```

Comment:
Braucht min. eine Zeile im Funktionskörper!

```suggestion
def game_over():
     return False
```